exp-player/addon/components/exp-lookit-story-page/component.js

```diff
@@ -336,6 +347,16 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
         }
     },
 
+    parentTextStyle: Ember.computed('parentTextBlock', function() {
+        var parentTextBlock = this.get('parentTextBlock') || {};
+        var css = parentTextBlock.css || {};
+        var cssString = '';
+        $.each(css, function(propName, value) {
+            cssString += new Ember.Handlebars.SafeString(`${propName}:  ${value};`);
```

Comment:
And thanks for the tip about providing `properties`. Should be set up for these two frames now.